ROADMAP.md

```diff
@@ -173,22 +173,67 @@ Any dates listed below and the specific issues that will ship in a given milesto
+* Incorporate versioning and upgrade design feedback [#1160](https://github.com/crossplane/crossplane/issues/1160)
+* Support for NoSQL database claims. Providers may now offer managed services that can be bound to this claim type. [#1356](https://github.com/crossplane/crossplane/issues/1356)
+* `KubernetesApplication` now supports:
+   - updates propagated to objects in a remote Kubernetes cluster. [#1341 ](https://github.com/crossplane/crossplane/issues/1341)
+   - sceheduling directly to a `KubernetesTarget` in the same namespace as a `KubernetesApplication`. [#1315 ](https://github.com/crossplane/crossplane/issues/1315 )
+* Experimental support for [OAM](https://oam.dev/) (Open Application Model) API types:
   * Revised [Kubernetes-friendly OAM spec](https://github.com/oam-dev/spec/pull/304/files)
   * OAM App Config Controller support [#1268](https://github.com/crossplane/crossplane/issues/1268)
   * Enhance Crossplane to support a choice of local and remote workload scheduling
-
-* Template Stacks 
-  * hardening and UX refinements
-  * [stack-minimal-gcp](https://github.com/crossplane/stack-minimal-gcp) enhancements for clean delete
-  * port [stack-minimal-aws](https://github.com/crossplane/stack-minimal-aws) and [stack-minimal-azure](https://github.com/crossplane/stack-minimal-azure) to Template Stacks
-
+* Security enhanced mode with `stack manage --restrict-core-apigroups`, which restricts packages from being installed with permissions on the core API group. [#1333 ](https://github.com/crossplane/crossplane/issues/1333)
 * Stacks Manager support for private repos and robot account credentials
 * Release process and efficiency improvements
 
+
+## v0.10.0
+* Backup/restore support - e.g. with Velero
+  - Allow a KubernetesApplication to be backed up and restored [crossplane#1382](https://crossplane/crossplane/issues/#1382)
+  - Allow connection secrets to be backed up and restored [crossplane-runtime#140](https://crossplane/crossplane-runtime/issues/#140)
+  - Support backup and restore of all GCP managed resources [provider-gcp#207](https://crossplane/provider-gcp/issues/#207)
+  - Support backup and restore of all Azure managed resources [provider-azure#128](https://crossplane/provider-azure/issues/#128)
+  - Support backup and restore of all AWS managed resources [provider-aws#181](https://crossplane/provider-gcp/issues/#181)
+  - Allow Stack, StackInstall, StackDefinition to be backed up and restored [crossplane#1389](https://crossplane/crossplane/issues/#1389)
+  - Backup and Restore doc [crossplane#1353](https://crossplane/crossplane/issues/#1353)
+
+* v1beta1 quality conformance doc [#933](https://github.com/crossplane/crossplane/issues/933)
+* v1beta1 quality for AWS API types  
+  - Networking and VPC [crossplane/provider-aws#145](https://github.com/crossplane/provider-aws/issues/145)
+  - S3 buckets [crossplane/provider-aws#99](https://github.com/crossplane/provider-aws/issues/99)
```

Comment:
Not super confident S3 buckets will be done in time for 0.10.0 due to their dependency on composition.

Reply:
moved out to roadmap, post 0.10.0